testing/kuttl/e2e/cluster-pause/files/01-pause-cluster.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: cluster-pause
+spec:
+  paused: true
+  instances:
+    - name: instance1
+      # We set replicas to 2, but this won't result in a new replica until we resume
+      replicas: 2
```

Comment:
Is there another change that we could make that wouldn't require creating a new pod? :thinking:

Reply:
I changed it from a replica bump to changing the type of service. Better?